frontend/src/app/features/details/summary/Summary.tsx

```diff
@@ -119,15 +120,22 @@ const Summary = () => {
   // State Initializations
   const initialParcelIds = [0];
 
-  const [location, setLocation] = useState([48.46762, -123.25458]);
-
   useEffect(() => {
     console.log('Change in details');
     let address = document.getElementsByTagName('h3');
     address.length > 0 && address[0] && address[0].remove();
     setEditSiteDetailsObject(details);
   }, [details]);
 
+  const [location, setLocation] = useState([48.46762, -123.25458]);
```

Comment:
do we need hard code value ?? 
i believe this will be different as per location.

Reply:
The values will be set as per the lat and long corresponding to each site

Reply:
Actually I don't need this, will remove it in next commit.